data/campaigns/The_Hammer_of_Thursagan/scenarios/02_Reclaiming_the_Past.cfg

```diff
@@ -15,32 +19,22 @@
 
     {DEFAULT_SCHEDULE}
 
-    {SCENARIO_MUSIC legends_of_the_north.ogg}
+    {SCENARIO_MUSIC revelation.ogg}
 
     [story]
         [part]
             {THOT_BIGMAP}
-            story= _ "Following the victory against the orcs, Aiglondur was summoned to the audience hall of Hamel, Lord of Knalga and Lord Companion of the Northern Alliance. Beside the dais stood a stranger in the robes of the Order of Loremasters..."
+            story= _ "Following the victory against the orcs, Aiglondur was summoned to the audience hall of Hamel, lord of Knalga and Lord Companion of the Northern Alliance. Beside the dais stood a stranger in the robes of the Order of Loremasters..."
```

Comment:
This reverts #9865.

Reply:
Ah, okay. If you haven't already, you might need to revisit the unit description changes then.